### PR TITLE
Use fullscreen element stack instead of fullscreen enabled flag.

### LINKED ISSUE #38: Fix references to HTML spec
[Elsewhere](http://lists.w3.org/Archives/Public/public-webapps/2014JulSep/0188.html), Paul Cotton noted that: 

> Section 6 [1] refers to bug 25310 [2] and suggests that the HTML5 sandboxing material could be changed. The current specification refers to HTML5.0 [3] which is currently in its last CR period before progressing to Proposed Recommendation. 
> If a change is required in the HTML specification then it is more likely that this would occur in HTML 5.1 [4]. 

I would strongly urge us to bypass all this and just reference the WHATWG HTML (i.e., [[!HTML]]).

### PR DESCRIPTION
**Pull request by mounirlamouri**
Fixes #38.

@annevk and @marcoscaceres WDYT?

### CODE REVIEW

**Comment by annevk** [comment]
Looks okay, though this does not cover the case where you would exit fullscreen through normal means.

File: index.html [hunk 1]
```diff
@@ -728,11 +728,11 @@
...
-          the top-level browsing document's <a>fullscreen enabled flag</a> has
-          been set. When that security condition applies, whenever the document
-          <a>fully exits fullscreen</a> and a screen orientation lock is
-          applied, the <a>user agent</a> MUST lock the orientation to the
-          default orientation.
+          the <a>top-level browsing context</a>'s <a>document</a>'s
+          <a>fullscreen element stack</a> is not empty. When that security
+          condition applies, whenever the document <a>fully exits fullscreen</a>
+          and a screen orientation lock is applied, the <a>user agent</a> MUST
+          lock the orientation to the default orientation.
```
Comment (marcoscaceres) [inline index.html]: Add cross-ref to `<a>default orientation</a>`

**Comment by mounirlamouri** [comment]
#annevk, is it better? I'm no longer using "fully exit fullscreen" but uses the fullscreen element stack being empty instead.

**Comment by mounirlamouri** [comment]
@annevk said it was okay on IRC even though it would be nicer with actual hooks. As long as those hooks are not in the fullscreen spec, I doubt we can do bettter here. Merging.

**Pull request merged**